sample storefront homepage

baltejas opened the pull request.

MallikarjunHt reviewed (approved):
nice design :+1: 

![Screenshot from 2020-10-09 11-04-01](https://user-images.githubusercontent.com/40291553/95546869-4a323380-0a1f-11eb-89b4-dcbfbae9870f.png)

baltejas commented:
@MallikarjunHt - I have updated the pull request with a new commit, I have removed my hardcoded code and I am now making use of magic scroll js for more robust design.
